sites/dotnet-guide/contents/tips/open-recent-projects/index.md

```diff
@@ -0,0 +1,22 @@
+---
+date: 2020-07-06
+title: Open Recent Projects
+technologies: [.net]
+products: [rider]
+topics: [navigation]
+author: khalidabuhakmeh
+subtitle: Quick ways to navigate to other solutions
+thumbnail: ./thumbnail.png
+cardThumbnail: ./card.png
+animatedGif:
+  file: './open-recent-projects.gif'
+  width: 500
+leadin: |
+    **Working with multiple projects?** 🥵
```

Comment:
Would go with a positive smiley here, multiple projects is not a bad or scary thing :-)